libs/core/src/lib/ui-events/ui-event.service.ts

```diff
@@ -7,6 +7,7 @@ import { WindowRef } from '../window/window-ref.service'
 @Injectable({ providedIn: 'root' })
 export class UIEventService {
   // Map<string, Map<FromEventTarget, Observable<UIEvent>>> -> FromEventTarget is internal source of rxjs
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
   private observables: Map<string, Map<any, Observable<UIEvent>>> = new Map()
```

Comment:
add thy type when creating the instance:
```
private observables = new Map<string, Map<any, Observable<UIEvent>>>()
```
then the `eslint-disable...` should not be needed